specification/resources/resource-manager/readme.java.md

```diff
@@ -18,7 +18,6 @@ Generate all API versions currently shipped for this package
 
 ``` yaml $(java) && $(multiapi)
 batch:
-  - tag: package-2019-06
```

Comment:
if there sdk released for this already? what's the impact of removing this line?

Reply:
I have added package-subscriptions-2019-06 below, so result is the same